oat-sa/extension-tao-itemqti
# Fx: indents in "graphicScoreMappingForm" tpl

DEPP: Answer score cannot be set for Select point interaction.

**Related to**

[AUT-9](https://oat-sa.atlassian.net/browse/AUT-9)

**How to test**

1. Go to Items tab.
2. Create a new item and click "Authoring".
3. Drag and drop Select point interaction to the canvas and choose the background image.
4. Click 'Response', draw any answer area on the interaction background and click on it.
5. No errors should appear
